_i18n/en.yml

```diff
@@ -418,7 +445,7 @@ help_pages:
       the login.gov confirmation email. Unfortunately, SAM will not be able to link
       your profile and information to your account.
     relink-my-profile: |-
-      If you cannot sign into your SAM account because you’ve permanently lost access to your security method and personal key, you can reset your account by deleting it.
+      If you cannot sign into your SAM account because you’ve permanently lost access to your security methods you can reset your account by deleting it.
```

Comment:
> permanently lost access to your security methods

Suggested:permanently lost access to your two-factor authentication methods

Reply:
Changed